README.md

```diff
@@ -1,3 +1,11 @@
+[![Go Reference](https://pkg.go.dev/badge/github.com/nginxinc/telemetry-exporter.svg)](https://pkg.go.dev/github.com/nginxinc/telemetry-exporter)
+[![Continuous Integration](https://github.com/nginxinc/telemetry-exporter/actions/workflows/ci.yml/badge.svg)](https://github.com/nginxinc/telemetry-exporter/actions/workflows/ci.yml)
+[![Go Report Card](https://goreportcard.com/badge/github.com/nginxinc/telemetry-exporter)](https://goreportcard.com/report/github.com/nginxinc/telemetry-exporter)
+![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/nginxinc/telemetry-exporter?logo=go)
+
```

Comment:
nit: could you remove empty lines for consistency? one is enough (as in the rest of this file)